src/main/java/gov/usgs/cida/pubs/busservice/sipp/SippConversionService.java

```diff
@@ -123,13 +119,10 @@ protected MpPublication buildObjectProperties(InformationProduct informationProd
 		mpPublication.setIpdsReviewProcessState(informationProduct.getTask());
 
 		String largerWorkTitle = informationProduct.getJournalTitle();
-		if (StringUtils.isNotBlank(largerWorkTitle)) {
-			if (PubsUtils.isPublicationTypeArticle(mpPublication.getPublicationType())
-					&& null != mpPublication.getPublicationSubtype()) {
-				mpPublication.setSeriesTitle(buildSeriesTitle(mpPublication.getPublicationSubtype(), largerWorkTitle));
-			} else {
+		if (StringUtils.isNotBlank(largerWorkTitle) &&
+			(!(PubsUtils.isPublicationTypeArticle(mpPublication.getPublicationType())
+					&& null != mpPublication.getPublicationSubtype()))) {
```

Comment:
Since they are all "and" conditions, there are unnecessary parenthesis.
Also, a null check is more important than method call check.
Suggested replacement...


> if ( StringUtils.isNotBlank(largerWorkTitle) 
>    && null != mpPublication.getPublicationSubtype() 
>    && ! PubsUtils.isPublicationTypeArticle(mpPublication.getPublicationType()) ) {

Reply:
Humm, it will not format as I expect.

Reply:
The refactored condition is not equivalent due to de Morgan’s Theorem: !(a && b) = !a || !b
A unit test caught it.  I agree that the conditional is ugly and hard to read. Left it as close to
the original to have the minimal chance of breaking. Having || in the conditional probably still leaves it hard to follow. Likely should be a helper method. Time wise, will probably leave as is.